Gemfile

```diff
@@ -20,7 +20,7 @@ gem 'uglifier', '>= 1.3.0'
 gem 'sprockets', '~> 3.0'
 
 # Application
-gem 'rails', '~> 4.2.2'
+gem 'rails', '~> 4.2'
```

Comment:
I don't want to fix the minor version because there is no reason to do so. I prefer more flexible dependencies.